lib/diagnostic.js

```diff
@@ -47,7 +48,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.call(alderaan);
```

Comment:
👍 
You need to return this in order to pass the tests.
`return deathStar.fire.call(alderaan);`